packages/caver-klay/caver-klay-accounts/src/index.js

```diff
@@ -109,39 +109,57 @@ Accounts.prototype.create = function create(entropy) {
     return this._addAccountFunctions(Account.create(entropy || utils.randomHex(32)));
 };
 
-Accounts.prototype.privateKeyToAccount = function privateKeyToAccount(privateKey, targetAddressRaw) {
-  var { privateKey: prvKey, address, isHumanReadable } = utils.parsePrivateKey(privateKey)
+// The key parameter can be either normal private key or klaytnWalletKey format.
```

Comment:
Please describe 
- what this function does
- Description of two parameters.

Reply:
Descriptions are added for each functions :)